test/rbacAPI.test.ts

```diff
@@ -12,21 +12,21 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import { newEnforcer } from '../src';
+import { newEnforcer, newRBACOperator } from '../src';
 
 it('test getImplicitRolesForUser', async () => {
   const e = await newEnforcer('examples/rbac_model.conf', 'examples/rbac_with_hierarchy_policy.csv');
-  expect(await e.getImplicitRolesForUser('bob')).toEqual([]);
-  expect(await e.getImplicitRolesForUser('alice')).toEqual(['admin', 'data1_admin', 'data2_admin']);
+  expect(await newRBACOperator(e).getImplicitRolesForUser('bob')).toEqual([]);
```

Comment:
The code in this page looks really bad. A lot of allocations like ``newRBACOperator(e)`` here. I don't think ``newRBACOperator()`` brings any benefit but lower performance and misunderstanding from users. Please give me pervasive reasons to do so if any.

Reply:
`newRBACOperator()` return RBAC method. 

Example:
````
const e = await newEnforcer('examples/rbac_model.conf', 'examples/rbac_with_hierarchy_policy.csv');

const rbac = newRBACOperator(e);

const roles  = rbac. getImplicitRolesForUser(e);
```